node/flatpak_node_generator/package.py

```diff
@@ -128,8 +128,10 @@ class LocalSource(PackageSource):
     path: str
 
 
-class Package(NamedTuple):
+@dataclass(frozen=True, eq=True)
+class Package:
     name: str
     version: str
     source: PackageSource
     lockfile: Path
+    install_path: Optional[str] = field(default=None)
```

Comment:
Do we actually need this field? I don't see it used anywhere. (If so, probably also just changing the default to `= None`, `field(...)` isn't really useful here.)

Reply:
No, not really. I've added it just for completeness.

Reply:
Personally I'd rather just remove it for now then, maybe make a comment note in `npm.py` that it's a field that exists that may be useful at some point? Otherwise it just looks like dead code.

Reply:
Ok, removed it.